deploy.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -e
 
-zip -FSr lambda.zip * -x terraform -x docs
+zip -FSr lambda.zip index.js *.json functions/ node_modules/
+npm install
```

Comment:
One minor change - let's change the order of these two lines. We want to install our libraries and then zip them.